package.json

```
@@ -24,12 +27,11 @@
     "tailwindcss": "4.1.3"
   },
   "devDependencies": {
+    "@biomejs/biome": "^2.3.6",
```

他の dependency と同様にバージョンを完全指定する形にしましょう！

```suggestion
    "@biomejs/biome": "2.3.6",
```

---

あー了解です🙇‍♂️